tests/test_ffmpeg_utils_unit.py

```diff
@@ -5,14 +5,14 @@
 
 
 def test_ffmpeg_seq_frame_vid_smokescreen() -> None:
-    """ """
+    """Test writing sequential video with `ffmpeg`."""
     image_prefix = "imgs_%d.jpg"
     output_prefix = "out"
     write_video(image_prefix, output_prefix)
 
 
 def test_ffmpeg_nonseq_frame_vid_smokescreen() -> None:
-    """ """
+    """Test writing non-sequential video with `ffmpeg`."""
```

Comment:
it actually is a sequential video, just the frames arent consecutive integers. More like they are ordered nanosecond timestamps. This requires different syntax in ffmpeg

Reply:
Updated.